docs/release-notes/release-2023-10-10.md

```diff
@@ -0,0 +1,99 @@
+---
+title: October 2023
+---
+
+# Release Notes October 2023 (2023-10-10)
+
+:::danger
+This release contains **breaking changes**
+:::
+
```

Comment:
````suggestion

## Typescript types breaking change
All packages updated to use typescript standard way of defining additional exports. This means packages consuming typespec must use `moduleResolution` setting with `node16` or `nodenext` in their tsconfig.json

You might see the following error 
```
 Cannot find module '@typespec/compiler/testing'. Did you mean to set the 'moduleResolution' option to 'nodenext'
```

This can be resolved by changing the following:

```diff
-"moduleResolution": "ESNext", 
+"moduleResolution": "node16", 
```

````

Reply:
For your consideration:

The `package.json` files for all packages have been changed to use the standard `types` field instead of the legacy `typesVersions` field to specify types for subpath exports. This means that packages consuming TypeSpec must use the `moduleResolution` setting with `node16` or `nodenext` in their `tsconfig.json`.